testing/btest/scripts/base/protocols/dns/tkey.zeek

```diff
@@ -0,0 +1,10 @@
+# @TEST-EXEC: zeek -b -C -r $TRACES/dns/tkey.pcap %INPUT > output
+# @TEST-EXEC: btest-diff dns.log
```

Comment:
Could add a `TEST-EXEC: test ! -f weird.log` to ensure the prior weird isn't generated anymore (an won't be in the future without noticing).